bzl/macros/buffham.bzl

```diff
@@ -127,6 +129,12 @@ buffham = macro(
             # Prevent receiving a `select` object on the input
             configurable = False,
         ),
+        "tags": attr.string_list(
+            default = [],
+            doc = "Tags to apply to the generated targets.",
+            # Prevent receiving a `select` object on the input
+            configurable = False,
```

Comment:
confused on why not using this now produces an error :/

w/e, didn't want a `select` here anyways